accord-core/src/test/java/accord/utils/Property.java

```diff
@@ -415,6 +413,8 @@ public static StatefulBuilder stateful()
     public static class StatefulBuilder extends Common<StatefulBuilder>
     {
         protected int steps = 1000;
+        @Nullable
+        protected Duration stepTimeout = null;
```

Comment:
`TopologyMixupTest` is what. uses this.  The test runs long running tasks that could block forever, so having a timeout per step will cause us to fail instead and recover the seed.